R/init.R

```diff
@@ -222,27 +186,246 @@ init <- function(data,
     )
   }
 
-  ns <- NS(id)
-  # Note: UI must be a function to support bookmarking.
-  res <- list(
-    ui = function(request) {
-      ui_teal(
-        id = ns("teal"),
-        modules = modules,
-        title = title,
-        header = header,
-        footer = footer
+  res <- list()
+  res$landing_popup_server <- NULL
```

Comment:
`res$landing_popup_server` is not needed as it can be passed directly to do.call down there ⬇️ 
```suggestion
```